jsonnet/components/cluster-monitoring-operator.libsonnet

```diff
@@ -150,6 +150,7 @@ function(params) {
       name: cfg.name,
       annotations: {
         'include.release.openshift.io/ibm-cloud-managed': 'true',
+        'include.release.openshift.io/hypershift': 'true',
```

Comment:
I'd be great if we're able to identify those CVO resources in `add-annotations.libsonnet` and inject the needed annotations there as well.

Reply:
Agree, I'll tackle the libsonnet cleanup on a different PR to unblock this one. Some files are not generated at all by the jsonnet and that's why they're left ignored. It'd be good to consolidate this.